.claude/skills/sync-opencode/SKILL.md

```
@@ -0,0 +1,152 @@
+mode: script-executor
+allowed-tools: Bash
+---
+
+# Sync OpenCode Skill (project-local)
+
+Deploys the generated OpenCode output at
+`target/opencode/` into an OpenCode config directory
+(default `~/.config/opencode/`) with the singular→plural directory
+rename that OpenCode expects. The pipeline is:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)